python/activity_stream/activity_stream.py

```diff
@@ -28,7 +28,6 @@
 shotgun_globals = sgtk.platform.import_framework(
     "tk-framework-shotgunutils", "shotgun_globals"
 )
-utils = sgtk.platform.import_framework("tk-framework-shotgunutils", "utils")
```

Comment:
Is this framework still used at all?

Reply:
In this repository, it was only used to call "utils.safe_delete_later()". I also reviewed if this function is used in other repositories but is used only here. The reason that it is deleted is that this function was deleted from "tk-framework-shotgunutils" also as part of Code cleanup for qt4

Reply:
Yes, that makes sense. 
My question was whether or not _tk-framework-shotgunutils_ was used somewhere else in qtwidgets. Because if not, we would cleanup the "dependency" in `info.yml`.
But I checked and yes,  _tk-framework-shotgunutils_  is still massively used in this repo so no need no cleanup.